api/src/messaging/serializers.py

```diff
@@ -1,6 +1,15 @@
 from common.abstract_serializers import ModelSerializerExtension
 from users.serializers import BasicUserSerializer
-from .models import Room, Message
+from .models import Room, Message, Membership
+
+from rest_framework import serializers
+
+# Membership serializer to relate the date joined to each member
+class MembershipSerializer(ModelSerializerExtension):
+    class Meta:
+        model = Membership
+        fields = "__all__"
+        exclude_fields = ["id"]
```

Comment:
Doesn't look like we use this serializer anywhere. We can remove it and undo the changes to this file